custom/fri/_design/views/extra_message/map.js

```diff
@@ -1,5 +0,0 @@
-function(doc) {
-    if(doc.doc_type === "FRIExtraMessage") {
-        emit([doc.domain, doc.message_id], null);
-    }
-}
```

Comment:
@dannyroberts do you recall what else should happen when we delete couch views?  Just wait for compaction to take care of it, or is some manual step or migration necessary?

Reply:
Great memory! That might have slipped my notice.

Yes. After deploying (and being sure we're not reverting!) you can run

```
cchq <env> django-mange prune_couch_views
```